src/ResourceManager/Sql/Commands.Sql/ServerActiveDirectoryAdministrator/Services/AzureSqlServerActiveDirectoryAdministratorAdapter.cs

```diff
@@ -65,10 +65,6 @@ public MicrosoftAzureCommandsResourcesModelsActiveDirectory.ActiveDirectoryClien
             {
                 if (_activeDirectoryClient == null)
                 {
-                    if (!Profile.Context.Environment.IsEndpointSet(AzureEnvironment.Endpoint.Graph))
```

Comment:
I added the check back in with the latest commit, and set the graph endpoint in the test framework.